code/modules/research/xenobiology/crossbreeding/warping

```diff
@@ -0,0 +1,796 @@
+	.= ..()
+	. += GLOB.resin_recipes
+
+
+
+
+/* pink rune, makes people happier when they get hugged on it */
+
+/obj/item/slimecross/warping/pink
+	colour = "pink"
+	effect_desc = "Draw a rune that hugs whoever steps on it.Only useful to friendless people."
+	runepath = /obj/effect/warped_rune/pinkspace
+
+/obj/effect/warped_rune/pinkspace
+	desc = "Love is the only reliable source of happiness we have left.But like everything, it comes with a price."
+	icon_state = "love_rune"
+	var/mob/living/carbon/human/loved
+
+
+/obj/effect/warped_rune/pinkspace/Initialize()
+	.=..()
+	RegisterSignal(T,COMSIG_ATOM_ENTERED,.proc/hug_rune)
+
+///adds the hug mood effect along with the sound effect.
+/obj/effect/warped_rune/pinkspace/proc/hug_rune()
+	for(var/mob/living/carbon/human/H in T)
+		playsound(T, "sound/weapons/thudswoosh.ogg", 50, TRUE)
+		SEND_SIGNAL(H, COMSIG_ADD_MOOD_EVENT, "hug", /datum/mood_event/hug)
+		H.visible_message(H, "<span class='notice'>You feel a magical embrace</span>")
+
+
+
+/*Gold rune : Turn things over it to gold, completely fucks over economy */
+
+/obj/item/slimecross/warping/gold
+	colour = "gold"
+	runepath = /obj/effect/warped_rune/goldspace
+	effect_desc = "draw a rune capable of turning nearly anything put on it to gold."
+
+/obj/effect/warped_rune/goldspace
+	icon_state = "midas_rune"
+	desc = "When everything is abundant,it becomes easy to forget when one had to work to obtain anything."
+
+/obj/effect/warped_rune/goldspace/Initialize()
+	.=..()
+	RegisterSignal(T,COMSIG_ATOM_ENTERED,.proc/midas_touch)
+
+///turn anything on the tile to a golden version of themselves. Sheets of any kind will be directly turned into gold bars.
+/obj/effect/warped_rune/goldspace/proc/midas_touch()
+	for(var/obj/item/stack/sheet/S in T)
+		if(istype(S,/obj/item/stack/sheet/mineral/gold)) //can't turn gold into gold
+			return
+		var/obj/item/stack/sheet/mineral/gold/G = new(T)
+		G.amount = rand(0, S.amount) //will give a random amount of gold between 0 and the amount of the original sheets
+		qdel(S)
+		return
+
+	for(var/obj/item/I in T)
+		I.material_flags = MATERIAL_COLOR | MATERIAL_ADD_PREFIX | MATERIAL_AFFECT_STATISTICS
+		I.set_custom_materials(list(/datum/material/gold=500)) 
+
+
+/*
+Anything after this is in """progress"""
+
+/*Any explosion over the rune will convert anything in the range of the explosion to SLIME, not people because even I'm not that evil */
+
+/obj/item/slimecross/warping/oil
+	colour = "oil"
+	runepath = /obj/effect/warped_rune/oilspace
+
+/obj/effect/warped_rune/oilspace
+	desc = "haha bomb machine go boom"
+
+/obj/item/slimecross/warping/black
+	colour = "black"
+	runepath = /obj/effect/warped_rune/blackspace
+
+/obj/item/slimecross/warping/lightpink
+	colour = "light pink"
+	runepath = /obj/effect/warped_rune/lightpinkspace
+
+/obj/item/slimecross/warping/adamantine
+	colour = "adamantine"
+	runepath = /obj/effect/warped_rune/adamantinespace
+
+/obj/item/slimecross/warping/rainbow
```

Comment:
suggestion: make this like the bluespace rune, but instead of swapping locations, it swaps (or transfers, if only one of the two mobs is sapient) minds (sort of like the recoil effect of a spent book of mind transfer).

Reply:
might need to restrict this to organic mobs only, just to prevent cheese with pAIs, minebot intelligence upgrades, posibrains, etc. alternatively, just cut out the "(or transfers, if only one of the two mobs is sapient)" bit.